docs/production/export-and-import.md

```diff
@@ -336,44 +362,31 @@ archive of all the organization's uploaded files.
-     be very careful to make sure the no users had their IDs
-     renumbered during the import process (this can be checked using
-     `manage.py shell` with some care). The push notifications
-     service has a mapping of which `user_id` values are associated
-     with which devices for a given Zulip server (represented by the
-     `zulip_org_id` registration). This means that if any `user_id`
-     values were renumbered during the import and you don't register a
-     new `zulip_org_id`, push notifications meant for the user who now
-     has ID 15 may be sent to devices registered by the user who had
-     user ID 15 before the data export (yikes!). The solution is
-     simply to not copy these settings and re-register your server for
-     mobile push notifications if any users had their IDs renumbered
-     during the logical export/import process.
-   - If you copy the `rabbitmq_password` secret from
-     `zulip-secrets.conf`, you'll need to run
-     `scripts/setup/configure-rabbitmq` as root to update your local RabbitMQ
-     installation to use the password in your Zulip secrets file.
-   - You will likely want to copy `camo_key` (required to avoid
-     breaking certain links) and any settings you added related to
-     authentication and email delivery so that those work on your new
-     server.
-   - Copying `avatar_salt` is not recommended, due to similar issues
-     to the mobile push notifications service. Zulip will
+   server, copying most of `/etc/zulip/settings.py` and
+   `/etc/zulip/zulip.conf` is safe and recommended, with the following
+   important exceptions and notes:
+
+   - Copying `avatar_salt` is not recommended. Zulip will
      automatically rewrite avatars at URLs appropriate for the new
      user IDs, and using the same avatar salt (and same server URL)
-     post import could result in issues with browsers caching the
-     avatar image improperly for users whose ID was renumbered.
+     post import could result in issues with browsers caching and
+     displaying avatar image improperly for users whose ID was
+     renumbered.
+   - Copying `zulip_org_id` and `zulip_org_key` is recommended to
+     avoid disconnecting your Zulip server from its registration with
+     the [Mobile Push Notifications Service][mobile-push].
+   - If you copy the `rabbitmq_password` secret from
+     `zulip-secrets.conf`, you'll need to run
+     `scripts/setup/configure-rabbitmq` as root to update your local
+     RabbitMQ installation to use the password in your Zulip secrets
+     file.
+   - Copying `camo_key` is required to avoid breaking links from Zulip
+     messages to externally hosted images.
+   - If your Zulip server is on an old Zulip Server release that
+     predates Zulip 5.0, and you use the [Mobile Push Notifications
+     Service][mobile-push], you should upgrade before you do the
+     export/import process if at all possible, and ask for support if
+     it is not.
```

Comment:
We should make "ask for support" some sort of link, either to CZO or sales@.

Reply:
Linked  to https://zulip.com/help/contact-support